### DISCUSSION TITLE
关于插件的一些疑问

### DISCUSSION THREAD

**Discussion by xingpingcn** [lobehub/lobehub#2837] [General]
在看了 #1399 后有点一头雾水，产生了几个疑问

1. 如果想要无障碍使用插件，是不是必须用docker部署？
2. 非openai的模型，在支持fc的情况下，例如gemini1.5pro，是不是也能通用其他插件？例如”实时天气“插件，我用vercel部署的lobe并不能使用该插件

**Comment by arvinxx** [comment]
> 如果想要无障碍使用插件，是不是必须用docker部署？

不是，其他部署的都可以


> 非openai的模型，在支持fc的情况下，例如gemini1.5pro，是不是也能通用其他插件？例如”实时天气“插件，我用vercel部署的lobe并不能使用该插件

理论上是的，不能的话可能有问题。可以提个issue 我看看
